troubleshooting/extended/post-issues.md

```diff
@@ -132,7 +132,7 @@ $ sudo AssetCacheManagerUtil activate
 AssetCacheManagerUtil[] Failed to activate content caching: Error Domain=ACSMErrorDomain Code=5 "virtual machine"...
 ```
 
-arise due to `VMM` flag being exposed by sysctl.
+arise due to `VMM` flag being exposed by sysctl, in addition to `kern.hv_vmm_present`, the latter only in mac OS Big sur 11.3 and higher mac OS versions.
```

Comment:
I'd probably change it to something like
"arise due to either the `VMM` flag being exposed by sysctl, or `kern.hv_vmm_present` in macOS Big Sur 11.3 and newer."

Current wording is a bit weird.

Reply:
Agree, sorry my english is terrible...
Not an expert on github also, is this ok or do I need to "resolve the conversation" too?
Thank you